### PR TITLE
Multi-DEM size function creation support

### PR DESCRIPTION
**Pull request by SorooshMani-NOAA**

### CODE REVIEW

File: geomesh/hfun/raster.py [hunk 1]
```diff
@@ -86,12 +86,20 @@ def __init__(self, raster: Raster, hmin: float = None, hmax: float = None,
...
 
+
         if window is None:
             iter_windows = list(self.iter_windows())
         else:
             iter_windows = [window]
 
         utm_crs = None
+
+        # NOTE: output_mesh is always in EPSG:4326 regardless of
```
Comment (jreniel) [inline geomesh/hfun/raster.py:97]: Is this still true? Shouldn't the output_mesh be the same CRS as the input raster?

Comment (JaimeCalzadaNOAA) [inline geomesh/hfun/raster.py:97]: Is this still true? Shouldn't the output_mesh be the same CRS as the input raster? @SorooshMani-NOAA

**Review by SorooshMani-NOAA — COMMENTED** [review]
I forgot to remove the comment, I'll take care of it. It returns in input raster CRS now. It used to return EPSG4326 before our previous dev discussion.

File: geomesh/hfun/raster.py (hunk 1 above)
Comment (SorooshMani-NOAA) [reply geomesh/hfun/raster.py:97]: I forgot to remove the comment, I'll take care of it. It returns in input raster CRS now. It used to return EPSG4326 before our previous dev discussion.

**Comment by JaimeCalzadaNOAA** [comment]
> I forgot to remove the comment, I'll take care of it. It returns in input raster CRS now. It used to return EPSG4326 before our previous dev discussion.

Great! I suspected that was the case, but just wanted to make sure.

**Comment by JaimeCalzadaNOAA** [comment]
Closed by mistake, reopened.

**Pull request merged**